dom/src/DOMSource.ts

```diff
@@ -8,7 +11,9 @@ export interface EventsFnOptions {
   preventDefault?: PreventDefaultOpt;
 }
 
-export interface DOMSource {
+export type DOMSource = MainDOMSource | DocumentDOMSource | BodyDOMSource; // | MockedDOMSource;
```

Comment:
Any reason why MockedDOMSource is not included? (it's okay if it's commented, as long as we write down the reason why it's excluded, in a comment)